TelCo.ColorCoder/ColorCode.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Text;
+
+namespace TelCo.ColorCoder
+{
+    internal static class ColorCode
```

Comment:
there are 2 classes ColorCode and ColorData, and methods are divided between these 2 classes.
The  intent of these classes are not clear from the naming.
This code organization can be improved!

Reply:
Sir, initially i had kept both the methods in same ColorPair class but due to algorithm line restriction(No of Line=50), it was not allowing to keep the project file structure >50 lines.